tests/integration/project/cmd_project_test.go

```diff
@@ -49,6 +49,17 @@ var _ = Describe("odo project command tests", func() {
 		})
 	})
 
+	Context("odo machine readable output on get project", func() {
+		It("should be able to get project", func() {
+			projectListJSON := helper.CmdShouldPass("odo", "project", "get", "-o", "json")
+			listOutputJSON, err := helper.Unindented(projectListJSON)
+			Expect(err).Should(BeNil())
+			partOfProjectListJSON, err := helper.Unindented(`{"kind":"Project","apiVersion":"odo.dev/v1alpha1","metadata":{"name":"` + project + `","namespace":"` + project + `","creationTimestamp":null},"spec":{},"status":{"active":true}}`)
+			Expect(err).Should(BeNil())
+			Expect(listOutputJSON).To(ContainSubstring(partOfProjectListJSON))
```

Comment:
I would say let's use `Expect(listOutputJSON).Should(MatchJSON(partOfProjectListJSON))` instead of ContainSubstring()

Reply:
oops, will update this